ci/tasks/light-aws/build.yml

```
@@ -0,0 +1,29 @@
+inputs:
+- name: builder-src
+- name: bosh-stemcells-ci
+- name: input-stemcell
+
+outputs:
+- name: light-stemcell
+
+run:
+  path: bosh-stemcells-ci/ci/tasks/light-aws/build.sh
+params:
+  bosh_io_bucket_name:        "bosh-aws-light-stemcells"
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)